[2023-11-20T08:43:30.541+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-16T00:00:00+00:00 [queued]>
[2023-11-20T08:43:30.546+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-16T00:00:00+00:00 [queued]>
[2023-11-20T08:43:30.547+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-20T08:43:30.554+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_data> on 2023-11-16 00:00:00+00:00
[2023-11-20T08:43:30.558+0000] {standard_task_runner.py:57} INFO - Started process 192 to run task
[2023-11-20T08:43:30.561+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'austin_animal_shelter_dag', 'load_data', 'scheduled__2023-11-16T00:00:00+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmput1a4bt6']
[2023-11-20T08:43:30.564+0000] {standard_task_runner.py:85} INFO - Job 46: Subtask load_data
[2023-11-20T08:43:30.599+0000] {task_command.py:416} INFO - Running <TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-16T00:00:00+00:00 [running]> on host c84b35a13244
[2023-11-20T08:43:30.641+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='austin_animal_shelter_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-16T00:00:00+00:00'
[2023-11-20T08:43:31.237+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/Scripts/etl.py", line 179, in load_data
    engine = create_engine(DATABASE_URL)
  File "<string>", line 2, in create_engine
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/util/deprecations.py", line 375, in warned
    return fn(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/create.py", line 514, in create_engine
    u = _url.make_url(url)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/url.py", line 738, in make_url
    return _parse_url(name_or_url)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/url.py", line 794, in _parse_url
    components["port"] = int(components["port"])
ValueError: invalid literal for int() with base 10: '{}'
[2023-11-20T08:43:31.241+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=austin_animal_shelter_dag, task_id=load_data, execution_date=20231116T000000, start_date=20231120T084330, end_date=20231120T084331
[2023-11-20T08:43:31.260+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 46 for task load_data (invalid literal for int() with base 10: '{}'; 192)
[2023-11-20T08:43:31.300+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-11-20T08:43:31.309+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-20T08:46:45.326+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-16T00:00:00+00:00 [queued]>
[2023-11-20T08:46:45.334+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-16T00:00:00+00:00 [queued]>
[2023-11-20T08:46:45.335+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-20T08:46:45.345+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_data> on 2023-11-16 00:00:00+00:00
[2023-11-20T08:46:45.350+0000] {standard_task_runner.py:57} INFO - Started process 199 to run task
[2023-11-20T08:46:45.354+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'austin_animal_shelter_dag', 'load_data', 'scheduled__2023-11-16T00:00:00+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpklkiyo49']
[2023-11-20T08:46:45.357+0000] {standard_task_runner.py:85} INFO - Job 45: Subtask load_data
[2023-11-20T08:46:45.406+0000] {task_command.py:416} INFO - Running <TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-16T00:00:00+00:00 [running]> on host d26cbef9e8ab
[2023-11-20T08:46:45.471+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='austin_animal_shelter_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-16T00:00:00+00:00'
[2023-11-20T08:46:48.262+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-20T08:46:48.273+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=austin_animal_shelter_dag, task_id=load_data, execution_date=20231116T000000, start_date=20231120T084645, end_date=20231120T084648
[2023-11-20T08:46:48.288+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-20T08:46:48.301+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-20T08:50:24.619+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-16T00:00:00+00:00 [queued]>
[2023-11-20T08:50:24.625+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-16T00:00:00+00:00 [queued]>
[2023-11-20T08:50:24.625+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-20T08:50:24.632+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_data> on 2023-11-16 00:00:00+00:00
[2023-11-20T08:50:24.636+0000] {standard_task_runner.py:57} INFO - Started process 189 to run task
[2023-11-20T08:50:24.638+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'austin_animal_shelter_dag', 'load_data', 'scheduled__2023-11-16T00:00:00+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpi_xt2yxe']
[2023-11-20T08:50:24.642+0000] {standard_task_runner.py:85} INFO - Job 46: Subtask load_data
[2023-11-20T08:50:24.677+0000] {task_command.py:416} INFO - Running <TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-16T00:00:00+00:00 [running]> on host 3ef39241d6e6
[2023-11-20T08:50:24.711+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='austin_animal_shelter_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-16T00:00:00+00:00'
[2023-11-20T08:50:27.762+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-20T08:50:27.772+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=austin_animal_shelter_dag, task_id=load_data, execution_date=20231116T000000, start_date=20231120T085024, end_date=20231120T085027
[2023-11-20T08:50:27.790+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-20T08:50:27.806+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-20T22:32:54.834+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-16T00:00:00+00:00 [queued]>
[2023-11-20T22:32:54.840+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-16T00:00:00+00:00 [queued]>
[2023-11-20T22:32:54.840+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-20T22:32:54.847+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_data> on 2023-11-16 00:00:00+00:00
[2023-11-20T22:32:54.852+0000] {standard_task_runner.py:57} INFO - Started process 247 to run task
[2023-11-20T22:32:54.859+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'austin_animal_shelter_dag', 'load_data', 'scheduled__2023-11-16T00:00:00+00:00', '--job-id', '128', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmp32znoouq']
[2023-11-20T22:32:54.862+0000] {standard_task_runner.py:85} INFO - Job 128: Subtask load_data
[2023-11-20T22:32:54.892+0000] {task_command.py:416} INFO - Running <TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-16T00:00:00+00:00 [running]> on host ad6c889005f6
[2023-11-20T22:32:54.938+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='austin_animal_shelter_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-16T00:00:00+00:00'
[2023-11-20T22:32:57.527+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-20T22:32:57.536+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=austin_animal_shelter_dag, task_id=load_data, execution_date=20231116T000000, start_date=20231120T223254, end_date=20231120T223257
[2023-11-20T22:32:57.584+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-20T22:32:57.598+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-20T22:48:01.523+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-16T00:00:00+00:00 [queued]>
[2023-11-20T22:48:01.530+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-16T00:00:00+00:00 [queued]>
[2023-11-20T22:48:01.530+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-20T22:48:01.539+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_data> on 2023-11-16 00:00:00+00:00
[2023-11-20T22:48:01.544+0000] {standard_task_runner.py:57} INFO - Started process 229 to run task
[2023-11-20T22:48:01.548+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'austin_animal_shelter_dag', 'load_data', 'scheduled__2023-11-16T00:00:00+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpz5jtsepx']
[2023-11-20T22:48:01.551+0000] {standard_task_runner.py:85} INFO - Job 55: Subtask load_data
[2023-11-20T22:48:01.583+0000] {task_command.py:416} INFO - Running <TaskInstance: austin_animal_shelter_dag.load_data scheduled__2023-11-16T00:00:00+00:00 [running]> on host fda681cc78c9
[2023-11-20T22:48:01.632+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='austin_animal_shelter_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-16T00:00:00+00:00'
[2023-11-20T22:48:04.992+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-20T22:48:05.016+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=austin_animal_shelter_dag, task_id=load_data, execution_date=20231116T000000, start_date=20231120T224801, end_date=20231120T224805
[2023-11-20T22:48:05.061+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-20T22:48:05.092+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
